lib/Search/CardSearchResultEntry.php

```diff
@@ -26,13 +26,22 @@
 
 namespace OCA\Deck\Search;
 
+use OCA\Deck\AppInfo\Application;
 use OCA\Deck\Db\Board;
 use OCA\Deck\Db\Card;
 use OCA\Deck\Db\Stack;
 use OCP\Search\SearchResultEntry;
 
 class CardSearchResultEntry extends SearchResultEntry {
 	public function __construct(Board $board, Stack $stack, Card $card, $urlGenerator) {
-		parent::__construct('', $card->getTitle(), $board->getTitle() . ' » ' . $stack->getTitle(), $urlGenerator->linkToRouteAbsolute('deck.page.index') . '#/board/' . $board->getId() . '/card/' . $card->getId(), 'icon-deck');
+		parent::__construct(
+			$urlGenerator->getAbsoluteURL(
+				$urlGenerator->imagePath(Application::APP_ID, 'card.svg')
```

Comment:
Wondering if this icon is a better match:

https://pictogrammers.com/library/mdi/icon/card-bulleted-outline/

Reply:
:+1: We had the same idea. It's the one I'm using in the reference widget.